<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenConfig</class>
 <widget class="QWidget" name="GenConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="blc_grp">
     <property name="title">
      <string>Black Level Correction</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="sensorBlack_lbl">
        <property name="text">
         <string>Sensor black level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="systemBlack_lbl">
        <property name="text">
         <string>System black level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="systemBlack"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sensorBlack0"/>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="sensorBlack1"/>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="sensorBlack2"/>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="sensorBlack3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
